x-pack/legacy/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/WatcherFlyout.tsx

```diff
@@ -157,57 +149,31 @@ export class WatcherFlyout extends Component<
   };
 
   public createWatch = () => {
-    const core = this.context;
     const { serviceName } = this.props.urlParams;
 
     if (!serviceName) {
       return;
     }
 
-    const emails = this.state.actions.email
-      ? this.state.emails
-          .split(',')
-          .map(email => email.trim())
-          .filter(email => !!email)
-      : [];
+    const email = this.state.actions.email ? this.state.emails : '';
```

Comment:
I've simplified this because we can just pass a csv to the action, because it uses `nodemailer` under the hood. Never tested it though.